storybook-testing/jest.config.js

```diff
@@ -9,7 +9,8 @@ module.exports = {
         '^.+\\.(scss|css)$': 'babel-jest'
     },
     transformIgnorePatterns: [
-        'node_modules\/?!(react-syntax-highlighter)'
+        'node_modules\/?!(react-syntax-highlighter)',
+        'node_modules/core-js(-pure|-compat)'
```

Comment:
Wasn't this in the other PR #1101?

Reply:
It is - I figured I'd merge that first but I needed this to be able to run the tests

Reply:
I think once this gets changes merged from master and the build succeeds, this is probably ready to go.